saleor/static/dashboard-next/products/urls.ts

```diff
@@ -16,17 +15,18 @@ export type ProductListUrlDialog =
   | "delete"
   | "save-search"
   | "delete-search";
-export type ProductListUrlFilters = Partial<{
-  isPublished: string;
-  priceFrom: string;
-  priceTo: string;
-  status: StockAvailability;
-  query: string;
-}>;
+export enum ProductListUrlFiltersEnum {
+  isPublished = "isPublished",
```

Comment:
Why `isPublished` is a string ?

Reply:
Because `qs` maps everything as `string`

Reply:
And... this is enum value. So basically this is what you see as a parameter name, in this particular situation `?query=example&isPublished=true` instead of `?1=example&4=true`.